src/components/breadcrumbs/crumb/crumb.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Crumb } from "./crumb.component";
+
+const LINK_TEXT = "Link text";
+
+describe("Crumb", () => {
+  it("passes href to the anchor element isCurrent is false", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} />);
+
+    const link = screen.getByText(LINK_TEXT);
+    expect(link).toBeInTheDocument();
+    expect(link.closest("a")).toHaveAttribute("href", "foo");
+  });
+
+  it("does not pass href to the anchor element when isCurrent is true", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} isCurrent />);
+
+    const anchor = screen.getByText(LINK_TEXT).closest("a");
+    expect(anchor).not.toHaveAttribute("href", "foo");
+
+    expect(anchor).toHaveStyle({
+      color: "var(--colorsYin055)",
+      textDecoration: "none",
+      cursor: "text",
+    });
+  });
+
+  it("calls the onClick callback if one is passed and isCurrent is false", async () => {
+    const onClickFn = jest.fn();
+
+    render(<Crumb href="foo" text={LINK_TEXT} onClick={onClickFn} />);
+    const link = screen.getByText(LINK_TEXT);
+    await userEvent.click(link);
+
+    expect(onClickFn).toHaveBeenCalledTimes(0);
```

Comment:
Since `handleClick` is the callback prop we want to provide, we'll need to update this test and its sad path counterpart below

```suggestion
  it("calls the handleClick callback if one is passed and isCurrent is false", async () => {
    const handleClickFn = jest.fn();

    render(<Crumb href="#" text={LINK_TEXT} handleClick={handleClickFn} />);
    const link = screen.getByText(LINK_TEXT);
    await userEvent.click(link);

    expect(handleClickFn).toHaveBeenCalledTimes(0);
```